dex/testing/loadbot/ts.go

```diff
@@ -0,0 +1,223 @@
+// This code is available on the terms of the project LICENSE.md file,
+// also available online at https://blueoakcouncil.org/license/1.0.0.
+
+package main
+
+import (
+	"errors"
+	"fmt"
+	"math/rand"
+	"strings"
+	"sync"
+	"time"
+
+	"decred.org/dcrdex/client/core"
+	"decred.org/dcrdex/dex/msgjson"
+	"decred.org/dcrdex/dex/order"
+)
+
+// ts erradicaly pushes the market one way or another randomly with huge buys
+// and sells moving the mid gap very quickly.
+type ts struct {
```

Comment:
I feel like the bot itself should be named **whale**.

Reply:
Ok. TS is a reference to the anonymous and malicious entity that tanks the DCR price, or TS (the seller).